src/room.cpp

```diff
@@ -37,8 +37,6 @@ void Room::Destroy() {
         }
         server = nullptr;
     }
-
-    room_thread->join();
```

Comment:
In case of RoomMember we'd still get confirmation that we successfully disconnected etc. (I believe)
I guess the same will hold true for Room.